include/client.h

```diff
@@ -549,6 +552,21 @@ class Client
         */
         void use_model_ensemble_prefix(bool use_prefix);
 
+        /*!
+        *   \brief Returns information about the given database node
+        *   \param address The address of the database node (host:port)
+        *   \returns parsed_map containing the database node information
+        */
+        parsed_map get_db_node_info(std::string address);
+
+        /*!
+        *   \brief Parse database node information from get_db_node_info()
+        *          into a nested unordered_map
+        *   \param info database node information
+        *   \returns parsed_map containing the database node information
+        */
+        parsed_map parse_db_node_info(std::string info);
```

Comment:
I wonder if this should be made into a private method ``_parse_db_node_info`` since it doesn't seem like a user would be calling it and it is currently called inside of ``get_db_node_info()``.

Reply:
yes I agree I'll switch it to private.